rich/logging.py

```diff
@@ -62,6 +63,7 @@ def __init__(
         *,
         show_time: bool = True,
         omit_repeated_times: bool = True,
+        limit_repeat_omissions: Union[int, float] = 1 / 2,
```

Comment:
Suggest the feature should be enabled with a boolean, and the 1/2 is not configurable.

`show_time_reminders` ?

Reply:
Agreed, changed to ````show_time_reminders: bool = True````